packages/fxa-content-server/app/scripts/views/settings.js

```diff
@@ -90,6 +91,7 @@ const View = BaseView.extend({
     context.set({
       ccExpired: !!this._ccExpired,
       escapedCcExpiredLinkAttrs: `href="/subscriptions" class="alert-link"`,
+      securityEventsVisible: this._securityEventsVisible,
```

Comment:
You can call `securityEventsVisible: this.displaySecurityEvents();` and remove the `this._securityEventsVisible` propery.